In [10]:
import data
import binaries
import matplotlib.pyplot as plt

## Finding EBs in the Villanova [Catalog](http://keplerebs.villanova.edu/)


Find the systems with orbital periods between 10.0 and 10.1 days.

In [6]:
kics = data.select_kics(pmin=10.0, pmax=10.1)
print kics

Found 3 systems in catalog meeting criteria.
[4840818 6775034 8358008]


## Retrieving and plotting data
Get the light curve data for one of the systems and plot.

In [7]:
time, flux, fluxerr, cadence, quarter, quality = data.loadlc_db(kics[1])

In [8]:
plt.plot(time[6000:9000], flux[6000:9000])
plt.ylim(0.94, 1.01)
plt.xlabel('Time (Days)')
plt.ylabel('Relative Flux')

## Model light curve

Try to make a model light curve that looks like the real data for KIC 4840818

In [9]:
eb = binaries.ModelBinary(sc=False)

NameError: name 'binaries' is not defined

In [ ]:
plt.plot(eb.time, eb.flux)
plt.xlim(0, 80)
plt.xlabel('Time (Days)')
plt.ylabel('Relative Flux')